test/CMakeLists.txt

```
@@ -301,6 +301,7 @@ else()
 endif()
 
 add_sdl_test_executable(checkkeys SOURCES checkkeys.c)
+add_sdl_test_executable(checkstruct SOURCES checkstruct.c)
```

This runs the test as part of CTest:
```suggestion
add_sdl_test_executable(checkstruct NONINTERACTIVE SOURCES checkstruct.c)
```